internal/services/formatters/generic/trivy/formatter_test.go

```diff
@@ -41,31 +41,46 @@ func TestTrivyParseOutput(t *testing.T) {
 		err = os.RemoveAll(".horusec")
 		assert.NoError(t, err)
 	})
-	t.Run("Should add 2 vulnerabilities on analysis without errors", func(t *testing.T) {
+	t.Run("Should add 153 vulnerabilities from FileSystemOutput and ConfigOutput on analysis without errors", func(t *testing.T) {
```

Comment:
Why do we need 153 vulnerabilities to test the parser?

Reply:
look real output now, before, a case that was not real was analyzed that had several treatment errors because of this

Reply:
I understand but we really need 153 vulnerabilities? The file seems really unnecessary large, we can't add a single result for every case?

Reply:
well, yeah, we can think of each case, but it will be a very complex and difficult job at the moment, creating a scenario with a real output of the tool solves our problem that we have here